src/test/java/org/apache/commons/lang3/ArrayUtilsTest.java

```diff
@@ -264,6 +266,31 @@ public int hashCode() {
             }
         }});
         assertEquals("bar", map.get("foo"));
+
+        // Return empty map when got input array with length = 0
+        assertEquals(Collections.emptyMap(), ArrayUtils.toMap(new Object[0]));
+
+        // Test all null values
+        map = ArrayUtils.toMap(new Object[][] { {null, null}, {null, null} });
+        assertEquals(new HashMap<Object, Object>() {
```

Comment:
Your expected map creation is too obtuse IMO, use the what Java provides instead:
```
Collections.singletonMap(null, null);
```

Reply:
Fixed. Thanks